features/steps/document.py

```diff
@@ -27,6 +27,11 @@ def given_a_blank_document(context):
     context.document = Document(test_docx('doc-word-default-blank'))
 
 
+@given('a Document object as document')
+def given_a_Document_object_as_document(context):
+    context.document = Document(test_docx('doc-default'))
```

Comment:
No need for a special document file yet, and better that we don't introduce one for this because bookmarks should work even when the document has none. We'll add your test file in a later commit.